logger/logger_test.go

```diff
@@ -59,6 +59,7 @@ func TestLogger_Log(t *testing.T) {
 		App:        "test",
 		IPAddress:  "127.0.0.1",
 		MacAddress: "C0:FF:EE:C0:FF:EE",
+		Timestamp:  time.Now(),
```

Comment:
@mdeltito I've added time.Now() to the first two tests (i.e. the case where the user provides timestamp as an option)- and not for the other tests. Let me know if you have different thoughts on how many tests to do this for.

Reply:
here you are adding it to the `Logger` options though, which doesn't really make sense. there isn't any case where a user should be configuring a timestamp globally.

because the new timestamp field doesn't make sense for `Logger` (unlike the rest) we may want to rethink `Options` slightly and consider it as an interface, with separate types for logger and message options. let's discuss.

Reply:
so a user should only be able to provide a timestamp when invoking LogWithOptions but not when instantiating a logger and providing options?

Reply:
also agree that it's odd conceptually to be attaching timestamp to options in the LogWithOptions case too

Reply:
https://github.com/logdna/logger-node/README.md

Reply:
https://github.com/logdna/logger-node/blob/master/README.md